modules.d/95nfs/module-setup.sh

```diff
@@ -120,8 +120,13 @@ install() {
     mkdir -m 0755 -p "$initdir/var/lib/nfs"
     mkdir -m 0755 -p "$initdir/var/lib/nfs/rpc_pipefs"
     mkdir -m 0770 -p "$initdir/var/lib/rpcbind"
-    [ -d "/var/lib/nfs/statd/sm" ] && mkdir -m 0755 -p "$initdir/var/lib/nfs/statd/sm"
-    [ -d "/var/lib/nfs/sm" ] && mkdir -m 0755 -p "$initdir/var/lib/nfs/sm"
+    [ -d "$dracutsysrootdir/var/lib/nfs/statd/sm" ] \
+        && mkdir -m 0700 -p "$initdir/var/lib/nfs/statd" \
+        && mkdir -m 0755 -p "$initdir/var/lib/nfs/statd/sm" \
+        && chown -R rpcuser:rpcuser "$initdir/var/lib/nfs/statd"
```

Comment:
`rpcuser` might be user for other distributions. 
Can we perhaps use `inst_dir` here instead which would pick up the correct user automatically from the host ?

Reply:
How do you mean to use `inst_dir`? To grep for permissions?

FTR this is not changing anything outside the generated initramfs. Maybe there would be better a way to detect that with grep, like here: https://github.com/dracut-ng/dracut-ng/blob/ed5ab5787177f2be8a620a8d2d63a9ad26fbf149/modules.d/95nfs/module-setup.sh#L137

WDYT?